testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/hibernate/sessionfactorytest/SessionFactoryTestCase.java

```diff
@@ -125,7 +124,6 @@ public void testInjectPCIntoHibernateSession() throws Exception {
 
     // Test that a Persistence unit can be injected into a Hibernate Session factory
     @Test
-    @Ignore // WFLY-17830: renable when upgrading to Hibernate ORM 6.2.1.Final which should have the https://github.com/hibernate/hibernate-orm/pull/6350 change included
```

Comment:
fyi re a smart thing Paul Ferraro does. Use @org.junit.Ignore and skip the import so you don't need to remember to remove it.  Which I forget half the time. ;)